docs/our_work/a_and_e_forecasting_tool.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
in mkdocs.yml there are some settings for categorizing the projects into the different work areas + whether theyre past or current projects, would you be able to add this project to that? It's what makes the navigation in the projects page nicer. Let me know if you dont get how to do it and I'll help!

Reply:
Done!